packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -401,7 +405,54 @@ export type InsertableTreeFieldFromImplicitField<
 	? ApplyKind<InsertableTreeNodeFromImplicitAllowedTypes<Types>, Kind, true>
 	: TSchema extends ImplicitAllowedTypes
 		? InsertableTreeNodeFromImplicitAllowedTypes<TSchema>
-		: unknown;
+		: never;
+
+/**
+ * {@inheritdoc (UnsafeUnknownSchema:type)}
+ * @alpha
+ */
+export const UnsafeUnknownSchema: unique symbol = Symbol("UnsafeUnknownSchema");
+
+/**
+ * A special value which some APIs accept as an alternative to a schema to indicate the schema can not easily be provided at compile time.
+ * @remarks
+ * When used this means the TypeScript typing should error on the side of completeness (allow all inputs that could be valid).
+ * This introduces the risk that out of schema data could be allowed at compile time, and only error at runtime.
+ *
+ * This only applies to APIs which input data which is expected to be in schema, since APIs outputting have easy mechanisms to do so in a type safe way even when the schema is unknown.
+ * In most cases that amounts to returning `TreeNode | TreeLeafValue`.
+ *
+ * This can be contrasted with the default behavior of TypeScript, which is to require the intersection of the possible types for input APIs,
+ * which for unknown schema defining input trees results in the `never` type.
+ * @privateRemarks
+ * Any APIs which use this must produce UsageErrors when out of schema data is encountered, and never produce unrecoverable errors,
+ * or silently accept invalid data.
+ * This is currently only type exported from the package: the symbol is just used as a way to get a named type.
+ * @alpha
+ */
```

Comment:
This is detailed documentation and I think it's useful - I also think we should add examples. If a user has trouble understanding what we mean by the explanations above, an example of the typing with and without the use of this token will make it immediately clear.

Reply:
Since the only current possible example using this is casting to ViewAlpha, and its types are still wrong even with that due to other variance issues, I don't think now is a great time to build out examples for this. This only really exists at this point to mitigate a regression in stuff we never intended to allow, and that use case is covered by the changeset in detail.

Once things are less broken and we consider more use-cases for this, then I think it would make sense to provide some examples, or at least link the APis using it which can have examples.

Reply:
If you have some reasonable example, you think including here would make sense feel free to provide one. As is I just don't see what would be useful and accurate that's worth including.

Reply:
We can wait until it's public then. I agree that having to do the casts is annoying and distracting in an example. I'm just trying to point out that if somebody comes across this type, they're probably going to be pretty confused as to what it does even after reading the doc comment. Not because it's a bad doc comment, but because the topic is inherently confusing. Slapping an example on that shows something like "if you use ImplicitFieldSchema, you get this on the API, but if you use UnsafeUnknownSchema, you get this instead" will make people go "OH I get the gist of this" with a lot less thinking.

Reply:
Sounds good to me!